### PR TITLE
feat: wrapper on portrait mode

### PR DESCRIPTION
**Pull request by SoleCincis**
- implemented a logic for big screens on portrait mode
  - based on maximum dimensions.width of mobile on portrait mode, according to: http://www.javascriptkit.com/dhtmltutors/cssmediaqueries2.shtml
- applied new dimension prop across the app
- in card list item when portrait big screen width should behave like on landscape

SVA-92

### CODE REVIEW

**Comment by SoleCincis** [comment]
@donni106 I brought whit me all this commits it means I made a mistake with my rebase. I'm not sure where this commits belongs to . I suspect that the cause is ; I did a git pull when I was already in my new branch feature/SVA-92-wrapper-text-on-portrait-for-big-screens . Btw the name of this branch is also wrong , right name is wrapper-on-portrait-for-big-screens. 
Sorry for the mess

File: src/components/CardListItem.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ const stylesWithProps = ({ horizontal, orientation, dimensions }) => {
 
   const width = horizontal
     ? imageWidth() * 0.7
-    : orientation === 'landscape'
+    : orientation === 'landscape' || (orientation === 'portrait' && dimensions.width > 450)
```
Comment (SoleCincis) [inline src/components/CardListItem.js:64]: the photos are cut to the right side , also on landscape mode. it seems that this problem is not generated by the new logic: 
Screenshot 2020-10-22 at 13 27 30

**Comment by SoleCincis** [comment]
current status : 
Screenshot 2020-10-22 at 12 16 12Screenshot 2020-10-22 at 12 16 25



Screenshot 2020-10-22 at 12 19 18Screenshot 2020-10-22 at 12 19 27

File: src/components/Wrapper.js [hunk 2]
```diff
@@ -31,15 +31,16 @@ export const WrapperWrap = styled(WrapperRow)`
 export const InfoBox = styled(WrapperRow)`
   margin-bottom: ${normalize(5)}px;
 `;
-
-export const WrapperWithOrientation = ({ orientation, children }) => {
-  if (orientation === 'landscape') {
+//dimensions.width
```
Comment (donni106) [inline src/components/Wrapper.js:34]: unnecessary comment

**Comment by donni106** [comment]
> @donni106 I brought whit me all this commits it means I made a mistake with my rebase. I'm not sure where this commits belongs to . I suspect that the cause is ; I did a git pull when I was already in my new branch feature/SVA-92-wrapper-text-on-portrait-for-big-screens . Btw the name of this branch is also wrong , right name is wrapper-on-portrait-for-big-screens.
> Sorry for the mess

ok thanks. i created a new branch based on latest master, cherry picked your commit, updated code with some todos and created a new pull request for that: https://github.com/ikuseiGmbH/smart-village-app-app/pull/111

**Pull request closed without merging**